lib/ansible/plugins/inventory/azure_rm.py

```diff
@@ -74,6 +74,13 @@
           type: bool
           default: False
           version_added: '2.8'
+        use_contrib_script_host_names:
```

Comment:
i would name it to 'plain_host_names' and add information about these not being unique and that when  `False` and additional 'hashing' is added to the name to ensure uniqueness.

Reply:
change made

Reply:
this has been done